pybluemonday error

**when i run python serve.py i get this error :** 
` * Importing gevent and monkey patching. Use --disable-gevent to disable.
Traceback (most recent call last):
  File "/home/kali/Documents/CTF_APP/CTFd/serve.py", line 21, in <module>
    from CTFd import create_app
  File "/home/kali/Documents/CTF_APP/CTFd/CTFd/__init__.py", line 19, in <module>
    from CTFd.plugins import init_plugins
  File "/home/kali/Documents/CTF_APP/CTFd/CTFd/plugins/__init__.py", line 9, in <module>
    from CTFd.utils.config.pages import get_pages
  File "/home/kali/Documents/CTF_APP/CTFd/CTFd/utils/config/pages.py", line 8, in <module>
    from CTFd.utils.security.sanitize import sanitize_html
  File "/home/kali/Documents/CTF_APP/CTFd/CTFd/utils/security/sanitize.py", line 1, in <module>
    from pybluemonday import UGCPolicy
ModuleNotFoundError: No module named 'pybluemonday'
`
**and when i install pybluemonday i get this error :** 
`Collecting pybluemonday==0.0.9
  Using cached pybluemonday-0.0.9.tar.gz (8.4 kB)
  Preparing metadata (setup.py) ... done
Requirement already satisfied: cffi~=1.1 in /usr/lib/python3/dist-packages (from pybluemonday==0.0.9) (1.15.1)
Building wheels for collected packages: pybluemonday
  Building wheel for pybluemonday (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [90 lines of output]
      uname_result(system='Linux', node='master', release='5.18.0-kali5-amd64', version='#1 SMP PREEMPT_DYNAMIC Debian 5.18.5-1kali6 (2022-07-07)', machine='x86_64')
      sh: 1: ./scripts/setup-linux-64.sh: not found
      make: *** No rule to make target 'clean'.  Stop.
      make: *** No rule to make target 'so'.  Stop.
      Requirement already satisfied: cffi~=1.1 in /usr/lib/python3/dist-packages (1.15.1)
      WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
      make: *** No rule to make target 'ffi'.  Stop.
      /usr/lib/python3/dist-packages/pkg_resources/__init__.py:123: PkgResourcesDeprecationWarning: 2.3.0-nmu1-b1 is an invalid version and will not be supported in a future release
        warnings.warn(
      /usr/lib/python3/dist-packages/setuptools/installer.py:27: SetuptoolsDeprecationWarning: setuptools.installer is deprecated. Requirements should be satisfied by a PEP 517 installer.
        warnings.warn(
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.linux-x86_64-cpython-311
      creating build/lib.linux-x86_64-cpython-311/pybluemonday
      copying pybluemonday/__init__.py -> build/lib.linux-x86_64-cpython-311/pybluemonday
      creating build/lib.linux-x86_64-cpython-311/tests
      copying tests/test_memory_leaks.py -> build/lib.linux-x86_64-cpython-311/tests
      copying tests/__init__.py -> build/lib.linux-x86_64-cpython-311/tests
      creating build/lib.linux-x86_64-cpython-311/tests/bluemonday
      copying tests/bluemonday/test_sanitize.py -> build/lib.linux-x86_64-cpython-311/tests/bluemonday
      copying tests/bluemonday/__init__.py -> build/lib.linux-x86_64-cpython-311/tests/bluemonday
      copying tests/bluemonday/test_policies.py -> build/lib.linux-x86_64-cpython-311/tests/bluemonday
      creating build/lib.linux-x86_64-cpython-311/tests/html_sanitizer
      copying tests/html_sanitizer/test_sanitize.py -> build/lib.linux-x86_64-cpython-311/tests/html_sanitizer
      copying tests/html_sanitizer/__init__.py -> build/lib.linux-x86_64-cpython-311/tests/html_sanitizer
      running build_ext
      $ GOPATH=/tmp/tmpzwg_6c60 go get -d
      go: go.mod file not found in current directory or any parent directory.
          'go get' is no longer supported outside a module.
          To build and install a command, use 'go install' with a version,
          like 'go install example.com/cmd@latest'
          For more information, see https://golang.org/doc/go-get-install-deprecation
          or run 'go help get' or 'go help install'.
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-56jc3cw7/pybluemonday_6895d2ff35124549be28fe130c51c0da/setup.py", line 49, in <module>
          setuptools.setup(
        File "/usr/lib/python3/dist-packages/setuptools/__init__.py", line 87, in setup
          return distutils.core.setup(**attrs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/core.py", line 185, in setup
          return run_commands(dist)
                 ^^^^^^^^^^^^^^^^^^
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/core.py", line 201, in run_commands
          dist.run_commands()
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/dist.py", line 973, in run_commands
          self.run_command(cmd)
        File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 1217, in run_command
          super().run_command(command)
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/dist.py", line 992, in run_command
          cmd_obj.run()
        File "/usr/lib/python3/dist-packages/wheel/bdist_wheel.py", line 299, in run
          self.run_command('build')
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/cmd.py", line 319, in run_command
          self.distribution.run_command(command)
        File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 1217, in run_command
          super().run_command(command)
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/dist.py", line 992, in run_command
          cmd_obj.run()
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/command/build.py", line 132, in run
          self.run_command(cmd_name)
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/cmd.py", line 319, in run_command
          self.distribution.run_command(command)
        File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 1217, in run_command
          super().run_command(command)
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/dist.py", line 992, in run_command
          cmd_obj.run()
        File "/usr/lib/python3/dist-packages/setuptools/command/build_ext.py", line 84, in run
          _build_ext.run(self)
        File "/usr/lib/python3/dist-packages/Cython/Distutils/old_build_ext.py", line 186, in run
          _build_ext.build_ext.run(self)
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/command/build_ext.py", line 346, in run
          self.build_extensions()
        File "/usr/lib/python3/dist-packages/Cython/Distutils/old_build_ext.py", line 195, in build_extensions
          _build_ext.build_ext.build_extensions(self)
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/command/build_ext.py", line 466, in build_extensions
          self._build_extensions_serial()
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/command/build_ext.py", line 492, in _build_extensions_serial
          self.build_extension(ext)
        File "/tmp/pip-install-56jc3cw7/pybluemonday_6895d2ff35124549be28fe130c51c0da/.eggs/setuptools_golang-2.3.0-py3.11.egg/setuptools_golang.py", line 154, in build_extension
          _check_call(cmd_get, cwd=pkg_path, env=env)
        File "/tmp/pip-install-56jc3cw7/pybluemonday_6895d2ff35124549be28fe130c51c0da/.eggs/setuptools_golang-2.3.0-py3.11.egg/setuptools_golang.py", line 111, in _check_call
          subprocess.check_call(cmd, cwd=cwd, env=dict(os.environ, **env))
        File "/usr/lib/python3.11/subprocess.py", line 413, in check_call
          raise CalledProcessError(retcode, cmd)
      subprocess.CalledProcessError: Command '('go', 'get', '-d')' returned non-zero exit status 1.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pybluemonday
  Running setup.py clean for pybluemonday                                                                                                                              
Failed to build pybluemonday
Installing collected packages: pybluemonday
  Running setup.py install for pybluemonday ... error
  error: subprocess-exited-with-error
  
  × Running setup.py install for pybluemonday did not run successfully.
  │ exit code: 1
  ╰─> [95 lines of output]
      uname_result(system='Linux', node='master', release='5.18.0-kali5-amd64', version='#1 SMP PREEMPT_DYNAMIC Debian 5.18.5-1kali6 (2022-07-07)', machine='x86_64')
      sh: 1: ./scripts/setup-linux-64.sh: not found
      make: *** No rule to make target 'clean'.  Stop.
      make: *** No rule to make target 'so'.  Stop.
      Requirement already satisfied: cffi~=1.1 in /usr/lib/python3/dist-packages (1.15.1)
      WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
      make: *** No rule to make target 'ffi'.  Stop.
      /usr/lib/python3/dist-packages/pkg_resources/__init__.py:123: PkgResourcesDeprecationWarning: 2.3.0-nmu1-b1 is an invalid version and will not be supported in a future release
        warnings.warn(
      /usr/lib/python3/dist-packages/setuptools/installer.py:27: SetuptoolsDeprecationWarning: setuptools.installer is deprecated. Requirements should be satisfied by a PEP 517 installer.
        warnings.warn(
      running install
      /usr/lib/python3/dist-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
        warnings.warn(
      running build
      running build_py
      creating build
      creating build/lib.linux-x86_64-cpython-311
      creating build/lib.linux-x86_64-cpython-311/pybluemonday
      copying pybluemonday/__init__.py -> build/lib.linux-x86_64-cpython-311/pybluemonday
      creating build/lib.linux-x86_64-cpython-311/tests
      copying tests/test_memory_leaks.py -> build/lib.linux-x86_64-cpython-311/tests
      copying tests/__init__.py -> build/lib.linux-x86_64-cpython-311/tests
      creating build/lib.linux-x86_64-cpython-311/tests/bluemonday
      copying tests/bluemonday/test_sanitize.py -> build/lib.linux-x86_64-cpython-311/tests/bluemonday
      copying tests/bluemonday/__init__.py -> build/lib.linux-x86_64-cpython-311/tests/bluemonday
      copying tests/bluemonday/test_policies.py -> build/lib.linux-x86_64-cpython-311/tests/bluemonday
      creating build/lib.linux-x86_64-cpython-311/tests/html_sanitizer
      copying tests/html_sanitizer/test_sanitize.py -> build/lib.linux-x86_64-cpython-311/tests/html_sanitizer
      copying tests/html_sanitizer/__init__.py -> build/lib.linux-x86_64-cpython-311/tests/html_sanitizer
      running build_ext
      $ GOPATH=/tmp/tmpzzuobd7_ go get -d
      go: go.mod file not found in current directory or any parent directory.
          'go get' is no longer supported outside a module.
          To build and install a command, use 'go install' with a version,
          like 'go install example.com/cmd@latest'
          For more information, see https://golang.org/doc/go-get-install-deprecation
          or run 'go help get' or 'go help install'.
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-56jc3cw7/pybluemonday_6895d2ff35124549be28fe130c51c0da/setup.py", line 49, in <module>
          setuptools.setup(
        File "/usr/lib/python3/dist-packages/setuptools/__init__.py", line 87, in setup
          return distutils.core.setup(**attrs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/core.py", line 185, in setup
          return run_commands(dist)
                 ^^^^^^^^^^^^^^^^^^
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/core.py", line 201, in run_commands
          dist.run_commands()
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/dist.py", line 973, in run_commands
          self.run_command(cmd)
        File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 1217, in run_command
          super().run_command(command)
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/dist.py", line 992, in run_command
          cmd_obj.run()
        File "/usr/lib/python3/dist-packages/setuptools/command/install.py", line 68, in run
          return orig.install.run(self)
                 ^^^^^^^^^^^^^^^^^^^^^^
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/command/install.py", line 698, in run
          self.run_command('build')
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/cmd.py", line 319, in run_command
          self.distribution.run_command(command)
        File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 1217, in run_command
          super().run_command(command)
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/dist.py", line 992, in run_command
          cmd_obj.run()
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/command/build.py", line 132, in run
          self.run_command(cmd_name)
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/cmd.py", line 319, in run_command
          self.distribution.run_command(command)
        File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 1217, in run_command
          super().run_command(command)
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/dist.py", line 992, in run_command
          cmd_obj.run()
        File "/usr/lib/python3/dist-packages/setuptools/command/build_ext.py", line 84, in run
          _build_ext.run(self)
        File "/usr/lib/python3/dist-packages/Cython/Distutils/old_build_ext.py", line 186, in run
          _build_ext.build_ext.run(self)
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/command/build_ext.py", line 346, in run
          self.build_extensions()
        File "/usr/lib/python3/dist-packages/Cython/Distutils/old_build_ext.py", line 195, in build_extensions
          _build_ext.build_ext.build_extensions(self)
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/command/build_ext.py", line 466, in build_extensions
          self._build_extensions_serial()
        File "/usr/lib/python3/dist-packages/setuptools/_distutils/command/build_ext.py", line 492, in _build_extensions_serial
          self.build_extension(ext)
        File "/tmp/pip-install-56jc3cw7/pybluemonday_6895d2ff35124549be28fe130c51c0da/.eggs/setuptools_golang-2.3.0-py3.11.egg/setuptools_golang.py", line 154, in build_extension
          _check_call(cmd_get, cwd=pkg_path, env=env)
        File "/tmp/pip-install-56jc3cw7/pybluemonday_6895d2ff35124549be28fe130c51c0da/.eggs/setuptools_golang-2.3.0-py3.11.egg/setuptools_golang.py", line 111, in _check_call
          subprocess.check_call(cmd, cwd=cwd, env=dict(os.environ, **env))
        File "/usr/lib/python3.11/subprocess.py", line 413, in check_call
          raise CalledProcessError(retcode, cmd)
      subprocess.CalledProcessError: Command '('go', 'get', '-d')' returned non-zero exit status 1.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> pybluemonday

note: This is an issue with the package mentioned above, not pip.
hint: See above for output from the failure.`

--- a/requirements.in
+++ b/requirements.in
@@ -28,5 +28,5 @@ WTForms==2.3.1
 python-geoacumen-city==2023.4.15
 maxminddb==1.5.4
 tenacity==6.2.0
-pybluemonday==0.0.10
+pybluemonday==0.0.11
 freezegun==1.2.2

--- a/requirements.txt
+++ b/requirements.txt
@@ -105,7 +105,7 @@ maxminddb==1.5.4
     #   python-geoacumen-city
 passlib==1.7.4
     # via -r requirements.in
-pybluemonday==0.0.10
+pybluemonday==0.0.11
     # via -r requirements.in
 pycparser==2.20
     # via cffi

